index.html

```diff
@@ -0,0 +1,123 @@
+<!DOCTYPE html>
+<html>
+<head>
+   <!-- Meta tags  -->
+   <meta charset="utf-8">
+   <meta http-equiv="X-UA-Compatible" content="IE=edge">
+   <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1">
+   <!-- End Meta tags  -->
+
+   <!-- Title  -->
+   <title>Software Developer Portfolio | Nwachan Theophile</title>
+   <!-- End Title  -->
+
+   <!-- Styles  -->
+   <link rel="stylesheet" href="css/style.css">
+
+   <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto">
+   <!-- Styles End  -->
+
+   <script src="https://kit.fontawesome.com/164f291a18.js" crossorigin="anonymous"></script>
+
+
+</head>
+<body>
+
+   <section id="header">
+      <header>
+         <h1 class="logo">NT</h1>
+         <nav>
+            <ul>
+               <li>
+                  <a href="#">
+                     <i class="fa-solid fa-bars"></i>
+                  </a>
+               </li>
+            </ul>

```

Comment:
- [ ] (OPTIONAL) It is not necessary to place your icon inside a `ul` and `li` tag since it is just one icon and not a list of icons.